js/rpg_core/Graphics.js

```diff
@@ -1199,7 +1199,7 @@ Graphics._switchFullScreen = function() {
  * @private
  */
 Graphics._isFullScreen = function() {
-    return ((document.fullScreenElement && document.fullScreenElement !== null) ||
+    return ((document.fullscreenElement && document.fullscreenElement !== null) ||
```

Comment:
I think this method is very confusing.
line 1202 means `Graphics._isFullScreen`,
but line 1203, 1204 means actually `Graphics._isNotFullScreen`.

Could you fix it? 🙇

Reply:
Ok. I simplified it to
```
 return document.fullscreenElement ||
           document.mozFullScreen || 
           document.webkitFullscreenElement ||
           document.msFullscreenElement;
```

Reply:
@bungcip 
I'm afraid this commit don't work.
Could you fix `Graphics._switchFullScreen`?

Reply:
ouch. after testing it with `npm run watch` and pressing `F4` I can confirm this commit don't work. I will change it tonight.